gateway/gateway.go

```diff
@@ -656,6 +660,15 @@ func (gateway *HandleT) getJobFromRequest(req *webRequestT) (job *jobsdb.JobT, n
 		"batch.0.context.sources.task_run_id",
 	).Str
 
+	jobData.version = gjson.GetBytes(
+		body,
+		"batch.0.context.library.name",
+	).Str + "/" +
+		gjson.GetBytes(
+			body,
+			"batch.0.context.library.version",
```

Comment:
Might be faster if we get `batch.0.context.library` only once and then get the name and version from it, something like:

```go
lib := gjson.GetBytes(body, "batch.0.context.library")
jobData.version = gjson.GetBytes(lib, "name")+gjson.GetBytes(lib, "version")
```